### PR TITLE
Emote redo + additions.

### PR DESCRIPTION
**Pull request by David-Stormwell**
## About The Pull Request

This is a re-do of a bunch of emotes! In this PR I'll be trying to do as many as I can, making them feel more smooth and flavorful, as well as adding any that should be or requested for the PR.
**Do note this is a draft, and I'm simply making this right now as a base.**

## Why It's Good For The Game

Flavorful emotes.

## Changelog
:cl:
add: Added more emotes!
add: Re-did emotes!
spellcheck: Will be fixing some grammar in some emotes that I saw in future commits.
/:cl:

### CODE REVIEW

**Comment by joookschad** [comment]
erp

**Comment by SpaceLoveSs13** [comment]
Please add a chocolate box and valentine's day card[like in tg]

**Comment by MLGTASTICa** [comment]
erp

File: code/modules/mob/living/carbon/xenomorph/emote.dm [hunk 1]
```diff
@@ -47,6 +47,10 @@
 	key = "hiss3"
 	sound = 'sound/voice/alien_hiss3.ogg'
 
+/datum/emote/living/carbon/xenomorph/kiss
+    key = "kiss"
+    key_third_person = "kisses"
+    message = "blows a kiss extending their claw outwards"
```
Comment (XSlayer300) [inline code/modules/mob/living/carbon/xenomorph/emote.dm:53]:
```suggestion
    message = "blows a kiss."
```

Comment (David-Stormwell) [reply code/modules/mob/living/carbon/xenomorph/emote.dm:53]: I'll change it again, but having a simple "Blows a kiss" is what I don't like about current emotes, far too bland and simple with no emotion or flavor.

Comment (David-Stormwell) [reply code/modules/mob/living/carbon/xenomorph/emote.dm:53]: Ping me again on discord if you really dislike it, but for now that's what I want to move away from

**Comment by bowlofcereal** [comment]
add valentine objectives, you won't

**Comment by David-Stormwell** [comment]
Shit, the bot reminded me this still exists

**Pull request closed without merging**